sdk/communication/communication-sms/review/communication-sms.api.md

```diff
@@ -14,6 +15,10 @@ export class SmsClient {
     constructor(connectionString: string, options?: SmsClientOptions);
     constructor(endpoint: string, credential: KeyCredential, options?: SmsClientOptions);
     constructor(endpoint: string, credential: TokenCredential, options?: SmsClientOptions);
+    // Warning: (ae-forgotten-export) The symbol "OptOutsClient" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    optOuts: OptOutsClient;
```

Comment:
Could you export the OptOutClient definition from index.ts?

Reply:
Done, thanks